packages/bpk-component-snippet/src/BpkSnippet.module.scss

```diff
@@ -0,0 +1,133 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use '../../unstable__bpk-mixins/breakpoints';
+@use '../../unstable__bpk-mixins/tokens';
+@use '../../unstable__bpk-mixins/typography';
+
+.bpk-snippet {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  @include breakpoints.bpk-breakpoint-small-tablet {
+    flex-direction: column;
+    gap: tokens.bpk-spacing-base();
+  }
+
+  &--row-reverse {
+    flex-direction: row-reverse;
+
+    @include breakpoints.bpk-breakpoint-small-tablet {
+      flex-direction: column;
+      gap: tokens.bpk-spacing-base();
+    }
+  }
+
+  &--vertical {
+    flex-direction: column;
+    gap: tokens.bpk-spacing-base();
+
+    &--container {
+      width: 100%;
+    }
+
+    &--content {
+      padding: 0;
+    }
+  }
+
+  &--container {
+    width: 50%;
+
+    @include breakpoints.bpk-breakpoint-small-tablet {
+      width: 100%;
+    }
+  }
+
+  &--image {
+    display: block;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+
+    // In Snippet component we’re sourcing only three options: 3:2 (landscape), square and 5:7 (portrait)
+    // https://www.figma.com/design/irZ3YBx8vOm16ICkAr7mB3/Backpack-Components?node-id=45681-58458&t=WIGrWPWvIuQYQKrk-0
+    &__landscape {
+      aspect-ratio: 3 / 2;
+    }
+
+    &__portrait {
+      aspect-ratio: 5 / 7;
+    }
+
+    &__square {
+      aspect-ratio: 1 / 1;
+    }
+
+    // rounded corner of the image
+    // https://skyscanner.slack.com/archives/C0JHPDSSU/p1721047356634579?thread_ts=1718608140.853189&cid=C0JHPDSSU
```

Comment:
We can't have internal links in our code

```suggestion
```

Reply:
Have removed!